celery/app/backends.py

```diff
@@ -37,6 +37,7 @@
     'consul': 'celery.backends.consul:ConsulBackend',
     'dynamodb': 'celery.backends.dynamodb:DynamoDBBackend',
     'azureblockblob': 'celery.backends.azureblockblob:AzureBlockBlobBackend',
+    'arangodb': 'celery.backends.arangodb.ArangoDbBackend'
```

Comment:
I think you need a `:` before `ArangoDbBackend` instead of a `.`.

Also, please add a trailing comma here.

Reply:
I didn't get the first comment

Reply:
I have added the trailing comma

Reply:
@dilipvamsi You have `celery.backends.arangodb.ArangoDbBackend` (everything separated by dots), but if you look at the other backends, the class-name (e.g. `ArangoDbBackend`) is separated by the module via a colon, so it should be `celery.backends.arangodb:ArangoDbBackend`. Subtle, but I believe it is what lets the dynamic importing work properly.